Documentation_displayed_on_the_portal/Share_Dataset_Instruction_Page.md

```diff
@@ -218,9 +218,11 @@ To add the newly created dataset to the list of CONP datasets present in the Dat
 - Send a pull request (PR) from your fork's `master` branch to the `master` branch of `https://github.com/CONP-PCNO/conp-dataset`. You should see 2 file changes in the PR:
   - Modification to the `.gitmodules` file to add the information for the new dataset. The added information should be of the form:
 
-  	```
-	[submodule "projects/<new_dataset_name>"]
-    	path = projects/<new_dataset_name>
-  		url = https://github.com/<username>/<new_dataset_name>.git
-  	```
+  		```
+		[submodule "projects/<new_dataset_name>"]
+			path = projects/<new_dataset_name>
+			url = https://github.com/<username>/<new_dataset_name>.git
+  		```
+   		*note: ensure that there is no empty lines at the end of the `.gitmodules` file otherwise it will not pass the format-checking tests of your PR.
+   
```

Comment:
Sorry, that's not quite what I meant.  There needs to be an empty line at the end of `.gitmodules` to pass the tests; the problem I was having was that new submodules were being added _after_ that empty line, and without an empty line after them.

Reply:
right. Should be corrected in the next commit. Thanks!